pkg/kubectl-argo-rollouts/cmd/get/get_rollout.go

```diff
@@ -235,3 +236,66 @@ func (o *GetOptions) PrintJob(w io.Writer, jobInfo info.JobInfo, prefix string,
 	name := o.colorizeStatus(jobInfo.Name, jobInfo.Status)
 	fmt.Fprintf(w, "%s%s %s\t%s\t%s %s\t%s\t%v\n", prefix, IconJob, name, "Job", o.colorize(jobInfo.Icon), jobInfo.Status, jobInfo.Age(), "")
 }
+
+func (o *GetOptions) setCanarySteps(roInfo *info.RolloutInfo) ([]string, int) {
+	newSteps := make([]string, 0)
+	currentIndex := 0
+	for k, step := range roInfo.Steps {
+		stepIndex := fmt.Sprintf("[%v]", k+1)
```

Comment:
I think we should be displaying the value of `stepIndex` and not `stepIndex+1`. For example, if there are three steps:

```yaml
steps:
- setWeight: 20
- pause: {}
- setWeight: 50
```

And it is still at the second step, then it would display:

```
[1]pause:∞ -> [2]setWeight: 50
```

Reply:
I tested it. It's OK  
yaml:
steps:
- setWeight: 20
- pause: {}
- setWeight: 50
- 
display:
Steps:         [1]setWeight:20 -> [2]pause:∞ -> [3]setWeight:50